ansible_events/websocket.py

```diff
@@ -1,22 +1,28 @@
 import json
 import logging
+from asyncio.exceptions import CancelledError
 
 import websockets
 
 
 async def send_event_log_to_websocket(event_log, websocket_address):
-    logger = logging.getLogger()
+    logger = logging.getLogger("ansible_events.websocket")
+    logger.info(f"websocket {websocket_address} connecting")
```

Comment:
Better to pass log message parameters separately and allow logging to format message for you:

```suggestion
    logger.info(f"websocket %s connecting", websocket_address)
```